charts/kubeai/values.yaml

```diff
@@ -30,6 +30,17 @@ modelServers:
       default: "fedirz/faster-whisper-server:latest-cpu"
       nvidia-gpu: "fedirz/faster-whisper-server:latest-cuda"
 
+modelServerPods:
+  # Security Context for the model pods
+  # Needed for OpenShift
+  securityContext:
```

Comment:
Just curious, no change needed. Have you seen this cause issues for pods?

I don't expect the LLM engines to fail but there is always a risk of issues when you enable more security settings by default. 

I do like having secure defaults that work! So I'm in favor of keeping this as-is.

Reply:
> Just curious, no change needed. Have you seen this cause issues for pods in general (not on KubeAI)?
> 
> I don't expect the LLM engines to fail but there is always a risk of issues when you enable more security settings by default.
> 
> I do like having secure defaults that work! So I'm in favor of keeping this as-is.

On OpenShift, from what I've observed - a container must set it's securityContext according to the privileges it requires so as to be assigned an appropriate SCC that matches it's securityContext settings. Failing to do so (for example, not setting securityContext at all or not matching the securityContext for the SCC which we are *hoping* to be assigned) will result in the default SCC to be assigned (which happens to be restricted-v2).

So, yeah this causing issues for pods is only true on OpenShift and not for others.

Reply:
Thank a lot for explaining. I get it now.